src/python/yapykaldi/asr.py

```diff
@@ -0,0 +1,161 @@
+from __future__ import (print_function, division, absolute_import, unicode_literals)
```

Comment:
Why the brackets?

Reply:
Just semantics. Useful when split over multiple lines.